src/sentry/static/sentry/app/views/projectDetail/projectTeamAccess.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Organization, Project} from 'app/types';
+import {SectionHeading} from 'app/components/charts/styles';
+import {t, tn} from 'app/locale';
+import Link from 'app/components/links/link';
+import space from 'app/styles/space';
+import Placeholder from 'app/components/placeholder';
+import IdBadge from 'app/components/idBadge';
+import Button from 'app/components/button';
+
+type Props = {
+  organization: Organization;
+  project?: Project | null;
+};
+
+type State = {
+  collapsed: boolean;
+};
+
+class ProjectTeamAccess extends React.Component<Props, State> {
+  state: State = {
+    collapsed: true,
+  };
+
+  static MAX_WHEN_COLLAPSED = 5;
+
+  onCollapseToggle = () => {
+    this.setState(prevState => ({
+      collapsed: !prevState.collapsed,
+    }));
+  };
+
+  renderInnerBody() {
+    const {project, organization} = this.props;
+
+    if (!project) {
+      return <Placeholder />;
```

Comment:
Do we have a size estimation we could use here to try and reduce page shift? Maybe pick the 75th% number of teams projects have?

Reply:
Yep, I'll be adjusting Placeholder heights once all sections of the sidebar are in place 👍